### ISSUE TITLE
RANSAC is not updating inlier ratio & hypothesis?

### ISSUE THREAD

**Issue by sunghoon031** [alejocb/dpptam#10]
I am reading your paper and investigated the code a bit. I love your method and the amazing result it produces !!  Here are some questions I have:

1. Do you update the inlier ratio for RANSAC anywhere in the code? Because I found that ransac_for_3Dspx function is not really updating the inlier ratio & hypothesis, but rather using a fixed number (i.e. 99) of hypotheses. 

2.  Correct me if I'm wrong: Is the published code "Semi-dense + 3DS (ours)" method? and is that the reason why the code doesn't involve the dense mapping part (section V.B in paper) ?

I will look forward to your reply.
Thanks in advance !

Cheers, 
Seong.

**Comment by alejocb** [comment]
Hi,

Thank you for your interest in the code!

1-) It seems that I do not update it, I used to do it but I changed it at some point. I released the code  3 months after the conference and it is possible that some parts are changed, although the main ideas remain the same. I will upload an improved version of the code very soon, and there will be some changes that are not in the IROS paper.

2-)  Yes.

Best regards,
Alejo.
